src/common/exceptions/http/service.unavailable.http.exception.ts

```diff
@@ -0,0 +1,23 @@
+import { LogContext, AlkemioErrorStatus } from '@common/enums';
+import { BaseHttpException } from './base.http.exception';
+import { HttpStatus } from '@nestjs/common';
+import { ExceptionDetails } from '@common/exceptions/exception.details';
+
+export class ServiceUnavailableHttpException extends BaseHttpException {
+  constructor(
+    message: string,
+    context: LogContext,
+    code: AlkemioErrorStatus | string = AlkemioErrorStatus.UNSPECIFIED,
```

Comment:
Again, this shouldn't be UNSPECIFIED imo, it's pretty clear that should be thrown when the service is unavailable.